### PR TITLE
[AC-5038-fix] -  fix backward compatibility import error

### PR DESCRIPTION
**Pull request by yotammanor**
**Test:**

_reproduce:_
- `git checkout AC-5038-mcproject; git pull`.
- on `cfg/base.cfg:107` set rev=AC-5038. 
- install the new revision: `sudo rm -rf src/django-accelerator/;sudo bin/buildout`
- run `bin/django makemigrations mc`, or `bin/django migrate`. See an import error.
  - **This will happen with any migration related command, with any app specified**

_fix_:
- on `cfg/base.cfg:107` set rev=AC-5038-fix. 
- install the new revision: `sudo rm -rf src/django-accelerator/;sudo bin/buildout`
- run `bin/django makemigrations mc`. See "no changes found" as expected.

### CODE REVIEW

File: simpleuser/migrations/0003_alter_username.py [hunk 1]
```diff
@@ -2,12 +2,16 @@
 # Generated by Django 1.10.3 on 2018-01-21 03:38
 from __future__ import unicode_literals
 
-import django.contrib.auth.validators
+try:
+    from django.contrib.auth.validators import UnicodeUsernameValidator
+except ImportError:  # pragma: no cover -  pre Django 1.10 compatibility
```
Comment (mc-nathan) [inline simpleuser/migrations/0003_alter_username.py]: This migration will never get run by a pre-Django 1.10.

**Comment by mc-nathan** [comment]
I do not understand under what conditions this fix is relevant.

**Comment by yotammanor** [comment]
@mc-nathan added clear reproduction instructions.

**Comment by yotammanor** [comment]
@mc-nathan decided to merge it in, since I think it'll be easier for me to work on it if I can test migration commands on mcproject. We can revisit this later as we go along.

**Pull request merged**